components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -55,6 +56,7 @@
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/profiles/profile.h"
 #if !defined(OS_ANDROID)
+#include "brave/ui/brave_custom_notification/message_popup_view.h"
```

Comment:
nit: `brave_custom_notification` seems excessively long and `brave_` is redundant. I guess what we name this depends on whether it is actually a generic custom notification or specific to ads